### PR TITLE
Lucy/fixing css issues

### PR DESCRIPTION
**Pull request by jkolyakov**
# Description

Changed styling files in frontend to [name].module.css. This stops stylings from different screens from overriding each other. Needed to change the styling calls to 'import styles from '.[filepath]'. Also refactored some in line styling in recover and save session.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Checked all pages for consistent styling

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by noorilu** [comment]
Do we merge this straight into main?

**Comment by jkolyakov** [comment]
> Did you mean to open the PR straight into main?

Whoops. I did not. My bad. Take a look at the changes to see the new styling when i re-open this though

File: mern/client/src/App.js [hunk 1]
```diff
@@ -3,9 +3,6 @@ import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
 import * as React from "react";
 
 import Navbar from "./Components/Navbar/navbar.component";
-import GetUser from "./Components/get-user.component";
```
Comment (anishalatchman) [inline mern/client/src/App.js:6]: tb to our early testing days... *sobbing emoji*

File: mern/client/src/Components/Buttons/ButtonStyleSheet.css [hunk 2]
```diff
@@ -32,21 +31,29 @@
...
 }
 
-.intent2{
+.intent2 {
   @apply h-24 bg-voiceflow-blue-light border-2 border-voiceflow-blue-light m-2 rounded-lg p-3 pl-6 pr-6 text-white font-nunito font-bold;
   width: 330px;
 }
 
-.intent3{
+.intent3 {
```
Comment (anishalatchman) [inline mern/client/src/Components/Buttons/ButtonStyleSheet.css:44]: love to see some consistent formatting 👍

File: mern/client/src/Components/Modals/GenericModal.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import "./ModalStyling.css";
+import GenericButton from "../Buttons/GenericButton";
+
+/**
+ * A generic input modal component
+ * @module GenericModal
+ */
+/** @prop {boolean} [show] : whether the modal shows or not*/
```
Comment (anishalatchman) [inline mern/client/src/Components/Modals/GenericModal.js:9]: question: what do these prop decorators do? is this just for documentation sake?

File: mern/client/src/Contexts/sessionProvider.js [hunk 4]
```diff
@@ -1,16 +1,17 @@
...
-
-
-    return (
-        <SessionContext.Provider value={[navState, setNavState]}>
-        { children }
-        </SessionContext.Provider>
-    );
+  // nav state to show or hide session id/save/delete buttons
+  const [navState, setNavState] = React.useState(false);
+  const [transcriptID, setTranscriptID] = React.useState();
```
Comment (anishalatchman) [inline mern/client/src/Contexts/sessionProvider.js:8]: future note: prolly don't need both sessionid and transcript id; can merge them

File: mern/client/src/screens/UploadTranscript/uploadTranscript.js [hunk 5]
```diff
@@ -80,13 +106,34 @@ function UploadTranscript() {
...
+            show={showModal}
+            title="Name your flow to begin"
+            body="Enter your flow name"
+            value={flowName.name}
+            onChange={handleFlowNameChange}
+            onClose={() => {
+              setShowModal(false);
+              deleteFile(transcriptID);
+            }}
+            onSubmit={() => {
```
Comment (anishalatchman) [inline mern/client/src/screens/UploadTranscript/uploadTranscript.js:125]: the uploadTranscript file has gotten a bit long (I know the generic components take up a good line count). is there anything we can extract in seperate files?

File: mern/client/src/utils/startScreen.js [hunk 6]
```diff
@@ -0,0 +1,24 @@
+import axios from "axios";
+
+var res;
+
+async function uploadFlow(flow) {
```
Comment (anishalatchman) [inline mern/client/src/utils/startScreen.js:5]: eventually need to move this file to frontend DAO once clean stucture is merged from my branch

**Review by anishalatchman — COMMENTED** [review]
Life-saving changes with the module.css files. TY! Just a few comments to make note of.

File: mern/client/src/Components/Modals/GenericModal.js (hunk 3 above)
Comment (jkolyakov) [reply mern/client/src/Components/Modals/GenericModal.js:9]: Yes. Documentation. If you hover over the Component tag it'll show you a description and the props (i.e. parameters that can go into it and what they do)

File: mern/client/src/Contexts/sessionProvider.js (hunk 4 above)
Comment (jkolyakov) [reply mern/client/src/Contexts/sessionProvider.js:8]: Noted

File: mern/client/src/screens/UploadTranscript/uploadTranscript.js (hunk 5 above)
Comment (jkolyakov) [reply mern/client/src/screens/UploadTranscript/uploadTranscript.js:125]: You're right. Need to explore splitting them into different components that are called in a new file

**Review by anishalatchman — APPROVED** [review]
Lookin hot. Just remember to look into extracting functions/if blocks into seperate files from uploadTranscript.js.

**Comment by noorilu** [comment]
> > Did you mean to open the PR straight into main?
> 
> Whoops. I did not. My bad. Take a look at the changes to see the new styling when i re-open this though

Yesss looking at the review and comments now, I'm assuming that my opened branch can be closed since this had all my changes as well? I agree with the uploadTranscript file being a lot longer than before. Will look into refactoring 👍

**Comment by noorilu** [comment]
Actually, neverminddd my pull request has some files that I made after this branch was made. It'll be a quick review though

**Pull request merged**